Code/GraphMol/Depictor/RDDepictor.cpp

```diff
@@ -261,7 +263,23 @@ void embedFusedSystems(const RDKit::ROMol &mol,
     for (auto rid : fused) {
       frings.push_back(arings.at(rid));
     }
-    EmbeddedFrag efrag(&mol, frings);
+
+    // don't allow ring system templates if >2 atoms in this ring system
+    // have a user-defined coordinate from coordMap
+    bool allowRingTemplates = useRingTemplates;
+    if (useRingTemplates && coordMap) {
+      std::unordered_set<unsigned int> coordMapAtoms;
+      for (const auto& ring : frings) {
+        for (const auto& aid : ring) {
+          if (coordMap->find(aid) != coordMap->end()) {
+            coordMapAtoms.insert(aid);
+          }
+        }
+      }
+      allowRingTemplates = (coordMapAtoms.size() < 2);
```

Comment:
Yeah, I originally was thinking it would be OK for someone to specify the placement of a bond within the template, so two atoms would be ok -- but I think <2 is the way to go because someone could easily give a bond length that isn't compatible with the templates. I will update this.